kinds/serviceaccount/serviceaccount_kind.cue

```diff
@@ -0,0 +1,46 @@
+package kind
+
+name:     "Serviceaccount"
```

Comment:
Should this be `ServiceAccount`?

Reply:
Yes, however that'll make codegen fail, I think it'll end up using a normalized `Serviceaccount` in some places and `ServiceAccount` in others. (I asked Eric the same thing as I was bugging him 🙈)

Reply:
Yes, as @sakjur said it failed the codegen... This is something we wanted to ask @sdboyer as well. :pray:

Reply:
I see, something for us to look into later :)